lib/detect-testing-library-utils.ts

```diff
@@ -644,6 +693,15 @@ export function detectTestingLibraryUtils<
         ) {
           importedCustomModuleNode = node;
         }
+
+        // check only if user-event import not found yet so we avoid
+        // to override importedUserEventLibraryNode after it's found
+        if (
+          !importedUserEventLibraryNode &&
+          String(node.source.value) === USER_EVENT_PACKAGE
+        ) {
+          importedUserEventLibraryNode = node;
```

Comment:
Since `userEvent` is imported from another package, I add to include its detection separately from the main Testing Library imported module.